multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -222,6 +231,17 @@ def create_fusion_data_processors(
             requires_label = False
             if data_types is not None and TEXT in data_types:
                 data_types.remove(TEXT)
+        elif per_name.lower().startswith(MMLAB_MODELS):
+            # create a multimodal processor for NER.
+            data_processors[ROIS].append(
+                create_data_processor(
+                    data_type=ROIS,
+                    config=config,
+                    model=per_model,
+                )
+            )
+            if data_types is not None and IMAGE in data_types:
+                data_types.remove(IMAGE)
```

Comment:
This `elif` seems unnecessary. Consider changing the mmdet_image's data_types in config and reusing the below logic to create a data processor.

Reply:
For now, following ner's implementation is fine. We can later refactor the creations of ner and detection processors since the current implementations are hacky.

Reply:
Besides, do we need `requires_label = False` here? I guess we don't need a separate label processor.

Reply:
Sure I'll remove the label processor in next pr, as well as the hacky implementation.

Reply:
We could change data_type to something like `mmlab_image` to support mmocr and mmseg and also remove this elif.

Reply:
I think we can still use `image` and check `problem_type` to determine which processor. (image processor or mmlab processor). `mmlab_image` may sound not intuitive for users.